src/Input/ToggleSwitch/ToggleSwitch.story.tsx

```diff
@@ -0,0 +1,81 @@
+import * as React from 'react';
+
+import ToggleSwitch from './ToggleSwitch';
+import { ToggleSwitchProps } from './ToggleSwitch.interface';
+import StorybookComponent from '../../../stories/StorybookComponent';
+import { SecondaryColor } from '../..';
+
+const onClick = (e: any) => {}
+
+const props: ToggleSwitchProps = {
+  defaultActive: false,
+  activeBallColor: SecondaryColor.white,
```

Comment:
Hmm we typically don't provide styling options unless the user is likely to want to change them. ([Point 4](https://github.com/glints-dev/glints-aries#code-procedure)). 

Another approach is to check with the designer whether they have some standard `theme` we can apply (like how we do it for the Button https://github.com/glints-dev/glints-aries/blob/v4/src/General/Button/Button.tsx#L14)

This comment applies to these props:
```
activeBackgroundColor,
inactiveBackgroundColor,
activeBallColor,
inactiveBallColor,
```

Reply:
Good insight!

I think for this, I will just default it to what I currently have from my designer, then I will iterate when the need arise.